extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/filters/accesslog/DefaultAccessLogReceiver.java

```diff
@@ -126,6 +126,14 @@ private DefaultAccessLogReceiver(final Executor logWriteExecutor, final Path out
         calculateChangeOverPoint();
     }
 
+    private String fixLogBaseName(String logBaseName) {
+        if(logBaseName.endsWith("."))
+        {
+        return logBaseName.substring(0,logBaseName.length()-1);
```

Comment:
would this workaround the issue?

@zhangjiangqige
@dmlloyd

Reply:
We would need a test that proves this